docs/cellxgene_census_schema.md

```diff
@@ -872,8 +881,185 @@ Cell metadata MUST be encoded as a `SOMADataFrame` with the following columns:
 </tbody>
 </table>
 
+### Census Spatial Sequencing Data – `census_obj["census_spatial_sequencing"][organism]` – `SOMAExperiment`
+
+Only Visium and Slide-seq are supported for spatial data. [See the "assays included" section above](assays).
+
+Spatial data for *Homo sapiens* MUST be stored as a `SOMAExperiment` in `census_obj["census_spatial_sequencing"]["homo_sapiens"]`.
+
+Spatial data for *Mus musculus* MUST be stored as a `SOMAExperiment` in `census_obj["census_spatial_sequencing"]["mus_musculus"]`.
+
+For each organism the `SOMAExperiment` MUST contain the following:
+
+* Cell metadata – `census_obj["census_spatial_sequencing"][organism].obs` – `SOMADataFrame`
+* Non-spatial data  –  `census_obj["census_spatial_sequencing"][organism].ms` – `SOMACollection`. This `SOMACollection` MUST only contain one `SOMAMeasurement` in `census_obj["census_spatial_sequencing"][organism].ms["RNA"]` with the following:
+  * Matrix  data –  `census_obj["census_spatial_sequencing"][organism].ms["RNA"].X` – `SOMACollection`. It MUST contain exactly one layers:
+    * Count matrix – `census_obj["census_spatial_sequencing"][organism].ms["RNA"].X["raw"]` – `SOMASparseNDArray`
+  * Feature metadata – `census_obj["census_spatial_sequencing"][organism].ms["RNA"].var` – `SOMAIndexedDataFrame`
+  * Feature dataset presence matrix – `census_obj["census_spatial_sequencing"][organism].ms["RNA"]["feature_dataset_presence_matrix"]` – `SOMASparseNDArray`
+* Obs to spatial data mapping:
+  * Obs to spatial data – `census_obj["census_spatial_sequencing"][organism].obs_spatial_presence`. It indicates the link between an observation and a scene, it MUST have three columns: 1) `soma_joinid` corresponding to `soma_joinid` of `obs` and 2) `scene_id` corresponding to the associated scene and 3) `data` which is a boolean value indicating presence.
+* Spatial data  –  `census_obj["census_spatial_sequencing"][organism].spatial` – `SOMACollection`.
+  * Spatial Scenes with spatial data  –  `census_obj["census_spatial_sequencing"][organism].spatial[scene_id]`  – `SOMAScene`.  There will be as many as Spatial Scenes as  spatial datasets. Each`SOMAScene` MUST contain the following:
+    * MUST contain a positions array – `census_obj["census_spatial_sequencing"][organism].spatial[scene_id].obsl["loc"]` – `PointCloudDataFrame`. This will contain the spatial array positions for each observation, the geometry points associated to them, and additional metadata.
```

Comment:
Editorial - This contains the spatial array positions for each observation, their associated geometry points, and additional metadata.